pkg/antctl/raw/check/installation/command.go

```diff
@@ -215,9 +215,9 @@ func (t *testContext) setup(ctx context.Context) error {
 			},
 		},
 		Tolerations: commonToleration,
-		Labels:      map[string]string{"app": echoSameNodeDeploymentName},
+		Labels:      map[string]string{"app": "antrea", "component": echoSameNodeDeploymentName},
```

Comment:
I have a slight preference for a "3-label" approach here with:
- `"app": "antrea"`
- `"component": "installation-checker"`
- `"name": echoSameNodeDeploymentName`

What do you think?

Reply:
That would be great. I think 2 out of 3 labels would be same for all 3 deployments ?

Reply:
Implemented. Thanks :)